MySQL.md

```diff
@@ -191,3 +197,32 @@ Here's typical wal-g configuration for that case:
 Restore procedure is straightforward:
 * start mysql (it's recommended to create new mysql instance)
 * fetch and apply desired backup using `wal-g backup-fetch "backup_name"`
+
+
+### MariaDB - using with `mariabackup`
+
+It's recommended to use wal-g with `mariabackup` tool in case of MariaDB for creating lock-less backups.
+Here's typical wal-g configuration for that case:
+```
+ WALG_MYSQL_DATASOURCE_NAME=user:pass@tcp(localhost:3305)/mysql                                                                                                                                      
+ WALG_STREAM_CREATE_COMMAND="mariabackup --backup --stream=xbstream --datadir=/var/lib/mysql"                                                                                                                               
+ WALG_STREAM_RESTORE_COMMAND="xbstream -x -C /var/lib/mysql"                                                                                                                       
```

Comment:
It shoud be `mbstream`, cause of [differences compared to xtrabackup](https://mariadb.com/kb/en/mariabackup-overview/#differences-compared-to-percona-xtrabackup) :)

Reply:
done